packages/cli/src/cli.ts

```diff
@@ -0,0 +1,45 @@
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+import { generate } from './commands/generate';
+
+export function cli(argv: string[]) {
+    return yargs(hideBin(argv)).command(
+        'generate',
+        'Generate a schema file from supplied database connection URL',
+        (yargs) => {
+            return yargs
+                .option('connectionString', {
+                    type: 'string',
+                    description:
+                        'Connection URL to the database to generate a schema from',
+                    alias: 'url',
+                    default:
+                        'postgres://postgres:postgres@localhost:5432/postgres',
+                })
+                .option('out', {
+                    type: 'string',
+                    description:
+                        'Output directory of the generated schema file',
+                    default: 'src/generated/synthql',
+                })
+                .option('defaultSchema', {
+                    type: 'string',
+                    description:
+                        'Default schema to be included in the generation',
+                })
+                .option('schemas', {
+                    description:
+                        'List of schemas to be included in the generation',
+                    default: [] as string[],
```

Comment:
Isn't this a bad default? Doesn't this mean that by default you include no schemas?

Reply:
Not exactly. The `defaultSchema` is always set, and its value is `public`. So the idea is, the `schemas` array should be set ONLY when you want to include more than one schema. Otherwise you should set the `defaultSchema` option, or use the default value for `defaultSchema` (i.e. `public`).